### PR TITLE
Update code-of-conduct to link to the CNCF's code-of-conduct

### PR DESCRIPTION
**Pull request by tuanvcw**
This PR updates code-of-conduct to link to the CNCF's code-of-conduct instead of copying it, so that Notary's code-of-conduct will be updated alongside with CNCF's code-of-conduct.

### CODE REVIEW

**Comment by tuanvcw** [comment]
Hi @justincormack,
Thanks for your review :)

Hi @cyli,
I'm just wondering if it's OK to move this PR forward?
If you have any concern, please don't hesitate to let me know.

Thank you all very much and have a great day :smile:

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,43 +1,82 @@
 ## CNCF Community Code of Conduct v1.0
```
Comment (thaJeztah) [inline CODE_OF_CONDUCT.md]: looks like the CNCF Code of Conduct was not updated yes? https://github.com/cncf/foundation/blob/master/code-of-conduct.md

> This Code of Conduct is adapted from the Contributor Covenant (http://contributor-covenant.org), version 1.2.0, available at http://contributor-covenant.org/version/1/2/0/

Should probably be updated there

**Comment by endophage** [comment]
Yeah, we're using the CNCF code of conduct as a member of the CNCF. We update when CNCF updates, not when the Contributor Covenant is updated.

@justincormack suggest we close this and address if/when CNCF updates.

**Comment by thaJeztah** [comment]
Would it be better to just _link_ to the CNCF code-of-conduct (https://github.com/cncf/foundation/blob/master/code-of-conduct.md) instead of copying it here?

**Comment by tuanvcw** [comment]
Hi @thaJeztah and @endophage, thanks for your kind reviews.
I've just updated the patch to just link to the CNCF code-of-conduct as suggestion from @thaJeztah.

**Pull request closed without merging**